ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -117,6 +119,46 @@ class DateTimePickerDemoController: DemoController {
         dateTimePicker.present(from: self, with: .dateRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType, titles: titles)
     }
 
+    @objc func presentPickerLeftToolbarView() {
+        let startDate = self.startDate ?? Date()
+        let endDate = self.endDate ?? Calendar.current.date(byAdding: .day, value: 1, to: startDate) ?? startDate
+        let titles: DateTimePicker.Titles
+        if datePickerType == .calendar && !UIAccessibility.isVoiceOverRunning {
+            titles = .with(startSubtitle: "Assignment Date", endSubtitle: "Due Date")
+        } else {
+            titles = .with(startTab: "Assignment Date", endTab: "Due Date")
+        }
+
+        let leftBarButtonItem = BarButtonItems.cancel(target: self, action: #selector(handleDidTapCancel))
+
+        dateTimePicker.present(from: self, with: .dateRange, startDate: startDate, endDate: endDate, datePickerType: datePickerType, titles: titles, leftBarButtonItem: leftBarButtonItem)
+    }
+
+    @objc func presentPickerLeftRightToolbarView() {
+        let startDate = self.startDate ?? Date()
```

Comment:
could you refactor this part of the code? only delta is adding the left and right bar button items

Reply:
can you have multiple items?

Reply:
Refactored creating an helper method for repeated code